src/Modal/Modal.stories.jsx

```diff
@@ -50,6 +51,11 @@ class ModalWrapper extends React.Component {
 ModalWrapper.propTypes = {
   title: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
   body: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
+  parentSelector: PropTypes.string,
+};
+
+ModalWrapper.defaultProps = {
+  parentSelector: '',
```

Comment:
I'm being uber-paranoid, but can we make the default value 'body' or something similar?

Reply:
<img width="739" alt="screen shot 2018-03-07 at 4 27 39 pm" src="https://user-images.githubusercontent.com/9357438/37119146-89a1e302-2224-11e8-8d0e-9ed37759b46e.png">
Seconded :)